proxy/proxy.go

```diff
@@ -1131,7 +1120,11 @@ func (p *Proxy) do(ctx *context) error {
 
 			p.metrics.IncErrorsBackend(ctx.route.Id)
 
-			if retryable(ctx.Request()) && perr.DialError() && ctx.route.BackendType == eskip.LBBackend {
+			if perr.code != 499 &&
+				retryable(ctx.Request()) &&
+				perr.DialError() &&
+				ctx.route.BackendType == eskip.LBBackend {
```

Comment:
Does it make sense to extract condition into a helper function like
```go
if retryable(ctx, perr) {
...
}
```
?

Reply:
great idea, done

Reply:
Now the test for `retryable` should be updated which is good as it may test that 499 is not retried.